pyproject.toml

```diff
@@ -1,34 +1,311 @@
+[project]
+name = "jwst"
+description = "Library for calibration of science observations from the James Webb Space Telescope"
+requires-python = ">=3.9"
+authors = [
+    { name = "JWST calibration pipeline developers" },
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
```

Comment:
Are all of these up to date with current settings?

Reply:
yes, I reran `peppyproject` against `master` just to be sure